README.rst

```diff
@@ -351,10 +416,126 @@ Run the pre-commit checks locally from an activated virtual environment with dev
 
     ./precommit.py  --overwrite
 
+Benchmarks
+----------
+
+Assuming that the virtual environment has been activated and the development
+dependencies have been installed, run the benchmarks with:
+
+.. code-block:: bash
+
+    ./benchmark/main.py *YOUR BUCKET*
+
+Some of our benchmark results:
+
+.. code-block:: text
+
+    Benchmark list 10000 files:
```

Comment:
are these benchmarks up to date? I think you copy-pasted something older, but other than that it looks good :)